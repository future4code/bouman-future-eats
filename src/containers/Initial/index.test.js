import React from 'react';
import renderer from 'react-test-renderer';
import { InitialPage } from './index'

test('initial page snapshot', () => {
    const tree = renderer
        .create(<InitialPage/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});