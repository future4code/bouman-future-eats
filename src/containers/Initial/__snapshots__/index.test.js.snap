// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initial page snapshot 1`] = `
<div
  className="sc-iAyFgw lkVrcV"
>
  <img
    alt="Logo Future Eats"
    className="sc-hSdWYo jbsCgH"
    src="logo-future-eats.png"
  />
</div>
`;
